Git-Discard-Unstaged-Changes.md

```diff
@@ -0,0 +1,13 @@
+### Discard unstaged changes
+
```

Comment:
Might want to spare a sentence on what _unstaged changes_ mean, or how could one end up in that situation. Take an example, to make your point even clearer.